.gitlab-ci.yml

```diff
@@ -139,14 +125,3 @@ Spack Test:
     - bash ci/tests/${TEST}/test.sh
   after_script:
     - whoami
-
-Spack Clean:
```

Comment:
Why is this being removed?

Reply:
We need to remove the spack build before we can build another one, instead of deleting after every time, it is now deleting before the build so that there is always an exaworks spack build available to test with.